google-play-book-downloader-pdf.py

```
@@ -194,11 +194,18 @@ def normalize_windows_cmd_caret(s: str) -> str:
 
+    cookies_unparsed = args.cookies
     cookies = {}
-    if not args.cookies:
+    if "Cookie" in headers:
+        if args.cookies:
+            logging.warning(f"Cookies appear in both --cookie argument and headers, check the cURL request in curl.txt")
+            cookies_unparsed.append(headers["Cookie"])
+        else:
+            cookies_unparsed = [headers["Cookie"]]
+    if not cookies_unparsed:
         logging.error(f"No cookies detected in curl.txt! You didn't properly copy the cURL request to curl.txt so the book will not be able to download correctly")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)